### DISCUSSION TITLE
Pharmacophore matching for 3D molecules

### DISCUSSION THREAD

**Discussion by SinaSarparast** [rdkit/rdkit#7772] [Q&A]
Hello,

I have conformations of a molecules. I have followed the guide [here](https://github.com/rdkit/UGM_2016/blob/master/Notebooks/Stiefl_RDKitPh4FullPublication.ipynb), but in this guide molecules are embedded into the 3d space for ph4 matching. How can I pass a molecule with existing 3d coordinates for pharmacophore matching? Can anyone help?

Thanks!

**Comment by AlceCarv** [comment]
I'm also looking for same solution, but my files are in mol2 from Sybyl-X
